### PR TITLE
PHP 8.4 support added

### PR DESCRIPTION
**Pull request by Atul-glo35265**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by Atul-glo35265** [comment]
This is blocked by [laminas/laminas-http](https://github.com/laminas/laminas-http/pull/84) which is not compatible with php8.4.

There is already a [PR](https://github.com/laminas/laminas-http/pull/84) for the same.
Once it is merged, tests will get passed and our PR can merge.

**Comment by fezfez** [comment]
@Atul-glo35265 You can simply ignore platform requirements, see https://github.com/fezfez/laminas-form/blob/d2c11882d6fe547c7d6217961fbc0469db3fbb34/.laminas-ci.json

**Comment by Atul-glo35265** [comment]
> @Atul-glo35265 You can simply ignore platform requirements, see https://github.com/fezfez/laminas-form/blob/d2c11882d6fe547c7d6217961fbc0469db3fbb34/.laminas-ci.json

Thanks @fezfez

File: docs/book/routing.md
Comment (samsonasik) [inline docs/book/routing.md:162]: why commit message is  "PHP 8.4 support added" and the change is unrelated to `.md` file https://github.com/laminas/laminas-router/pull/63/commits/513803eca4e3ea1069c4d6591e00c2c204c1264b ?  We don't use squash on merge so I think it would be better to have proper message if this is a blocker for latest CI.

Comment (froschdesign) [reply docs/book/routing.md:162]: An example of good commit messages can be found in this pull request: https://github.com/laminas/laminas-feed/pull/89

Comment (Atul-glo35265) [reply docs/book/routing.md:162]: I've updated the commit message.
@samsonasik I'll take care of it next time. Thanks @froschdesign for the reference.

**Comment by Atul-glo35265** [comment]
@gsteel, @samsonasik,
Please review this PR and do needful.

**Review by gsteel — APPROVED** [review]
Thanks @Atul-glo35265

**Pull request merged**
